docs/docs/README.md

```diff
@@ -0,0 +1,21 @@
+# Documentation Area
```

Comment:
This file describes, how developers can use docusaurus, to write docs on their local machine. We dont need this in /docs/docs (which is also a weird path :D )